src/Illuminate/Support/Fluent.php

```diff
@@ -58,7 +58,7 @@ public function getAttributes()
     /**
      * Convert the fluent instance to an array.
      *
-     * @return array
+     * @return array|string
```

Comment:
This doesn't makes any sense. It's always going to be an array.

Reply:
The `jsonSerialize()` method returns `toArray()`. So when the `jsonSerialize()` method works, the string `json` is now stored in the `$attributes` array. As such, `toArray()` and `jsonSerialize()' rarely return a string. It was not meant to be used that way. I will find a solution and send a pull request regarding this issue ...

Reply:
@driesvints The tests drove me to this idea, now I realized it was about the test situation. I'm pulling this part back.